pkg/components/prometheus-operator/component_test.go

```diff
@@ -89,6 +91,52 @@ component "prometheus-operator" {
     }
   }
 }
+`,
+			wantErr: false,
+		},
+		{
+			desc: "prometheus operator and admission webhook toleration are set correctly",
+			hcl: `
+component "prometheus-operator" {
+  operator {
+    admission_webhook_tolerations {
+      key      = "lokomotive.io/operator-admission-webhook"
+      operator = "Equal"
+      value    = "test"
+      effect   = "NoSchedule"
+    }
+    tolerations {
+      key      = "lokomotive.io/operator"
+      operator = "Equal"
+      value    = "test"
+      effect   = "NoSchedule"
+    }
+  }
+}
```

Comment:
How about a conversion test to test that these values are getting on to the actual config at the Kubernetes level.

Reply:
Does not have to be a part of this function. It could be a standalone test to verify all the changes that went in this PR.

Reply:
@surajssd Can you please link some examples of such tests to make it easier for @knrt10 to implement them?

Reply:
I think I understand what Suraj is trying to say. I will implement it